activity.py

```diff
@@ -103,14 +103,13 @@ def _invalid_number_alert(activity):
 
 
 def _extract_value(value):
-    try:
+    if isinstance(value, str):
         a,b = value.split('/')
-        value = float(a)/float(b)
-    except ZeroDivisionError:
-        value = 0
-    except ValueError:
-        value = float(Value)
-
+        try:
```

Comment:
Why not test for zero and raise a ZeroDivisionError in that case?

Reply:
I'm really sorry if I'm misunderstanding something, but I feel that they do the same thing. Is there any particular reason to avoid the try-except block?

Reply:
In general, exceptions are not considered good practice for simple flow control. They're typically reserved for when you don't know how to handle a situation. In this case, you know you have to take 0 values into account, so you might as well handle it yourself.

Reply:
@aguzubiaga Thanks for explaining! I've made a new commit, could you please take a look?